server/plugin/utils.go

```diff
@@ -384,3 +386,22 @@ func lastN(s string, n int) string {
 
 	return string(out)
 }
+
+func getOrganizations(organizations string) ([]string, map[string]bool) {
```

Comment:
That is the usecase of also returning a `map`. It seems to over complicated the code.

Reply:
This seems unresolved.